### ISSUE TITLE
copy_model_from loses comments under $PROB

### ISSUE THREAD

**Issue by kylebaron** [metrumresearchgroup/bbr#372]
# Summary
As a user, I want `copy_model_from()` to copy all comments from the old control stream to the new control stream

# Tests

**Comment by seth127** [comment]
We talked about this awhile ago (see [this comment](https://github.com/metrumresearchgroup/bbr/pull/256#discussion_r512187065) for some context). Either way, you can opt out of this with `.update_model_file = FALSE`. See [code here](https://github.com/metrumresearchgroup/bbr/blob/develop/R/copy-model-from.R#L29).

Does that satisfy what this issue is looking for?

**Comment by kylebaron** [comment]
https://github.com/metrumresearchgroup/bbr/blob/develop/R/copy-model-from.R#L31-L33

I do want to update; but I read this to mean that only the `$PROBLEM` statement would be modified.

**Comment by kylebaron** [comment]
Just clarified the title; I'm talking about lines that are commented out (with `;`), underneath $PROBLEM

**Comment by seth127** [comment]
Right, you can see in [the code here](https://github.com/metrumresearchgroup/bbr/blob/develop/R/copy-model-from.R#L189) (specifically the regex on line 196) that we cut everything between `$PROB` and the next `$`. Maybe that's the wrong thing to do? 

My understanding was that those comments under `$PROB` were sort of like an "extended problem statement" and so they should be blown away to avoid them persisting through to models that they don't apply to down the line. Basically the idea was that we want to encourage people to use `tags` and `notes` for this kind of info instead of putting it in multiline comments in the control stream that would need to be updated as the model evolves. 

Does that make sense?
